openforcefield/tests/test_molecule.py

```diff
@@ -52,64 +53,124 @@ def assert_molecule_is_equal(molecule1, molecule2, msg):
+    for atom in torsion:
+        for bond in atom.bonds:
+            # Consider the bond only if both atoms are in the torsion.
+            if (bond.atom1_index in torsion_atom_indices and
+                        bond.atom2_index in torsion_atom_indices):
+                bonds_by_atom_idx[bond.atom1_index].add(bond.atom2_index)
+                bonds_by_atom_idx[bond.atom2_index].add(bond.atom1_index)
+
+    # Find the central atom, which is connected to all other atoms.
+    atom_indices = [i for i in torsion_atom_indices if len(bonds_by_atom_idx[i]) == 3]
+    if len(atom_indices) != 1:
+        return False
+    central_atom_idx = atom_indices[0]
+
+    # Find the atom outside the ring.
+    atom_indices = [i for i in torsion_atom_indices if len(bonds_by_atom_idx[i]) == 1]
+    if len(atom_indices) != 1 or central_atom_idx not in bonds_by_atom_idx[atom_indices[0]]:
+        return False
+    outside_atom_idx = atom_indices[0]
+
+    # Check that the remaining two atoms are non-central atoms in the membered ring.
+    atom1, atom2 = [i for i in torsion_atom_indices if i not in [central_atom_idx, outside_atom_idx]]
+    # The two atoms are bonded to each other.
+    if atom2 not in bonds_by_atom_idx[atom1] or atom1 not in bonds_by_atom_idx[atom2]:
+        return False
+    # Check that they are both bonded to the central atom and none other.
+    for atom_idx in [atom1, atom2]:
+        if (central_atom_idx not in bonds_by_atom_idx[atom_idx] or
+                    len(bonds_by_atom_idx[atom_idx]) != 2):
+            return False
+
+    # This is a torsion including a three-membered ring.
+    return True
+
+
+#=============================================================================================
+# FIXTURES
+#=============================================================================================
 
-        self.molecules = copy.deepcopy(TestMolecule.test_molecules)
+def mini_drug_bank():
+    """Load the full MiniDrugBank into Molecule objects."""
+    # If we have already loaded the data set, return the cached one.
+    if mini_drug_bank.molecules is not None:
+        molecules = mini_drug_bank.molecules
+    else:
+        # Load the dataset.
+        file_path = get_data_filename('molecules/MiniDrugBank_tripos.mol2')
+        molecules = Molecule.from_file(file_path, exception_if_undefined_stereo=False)
+        molecules = [mol for mol in molecules if mol is not None]
+        mini_drug_bank.molecules = molecules
+    return molecules
+
+# Use a "static" variable as a workaround as fixtures cannot be
+# used inside pytest.mark.parametrize (see issue #349 in pytest).
+mini_drug_bank.molecules = None
+
+
+#=============================================================================================
+# TESTS
+#=============================================================================================
+
+@pytest.mark.skipif(not OpenEyeToolkitWrapper.toolkit_is_available(), reason=_OPENEYE_UNAVAILABLE_MESSAGE)
```

Comment:
To get you up to speed -- John and I have talked about getting test molecules into a "neutral" format (likely serialized OFF `Molecule`s). Right now our OFF `Molecule` representation isn't totally stable, but hopefully in the long run this dependence on OE for loading datasets will go away.